MyCapytain/resources/texts/local/capitains/cts.py

```diff
@@ -248,6 +248,11 @@ def getValidReff(self, level=None, reference=None, _debug=False):
                 message = ", ".join(duplicates)
                 warnings.warn(message, DuplicateReference)
             del duplicates
+            empties = [n for n in passages if n.strip('.') != n or n == '']
+            print(passages)
```

Comment:
Probably something that made it to the final commit that should not have ? :)

Reply:
I will remove that.